File: bird.py
n = input()
arr = map(int, raw_input().split(" "))
arr.sort()
count_arr = list()
for x in range(0,5):
    count_arr[x] = 0
for x in range(0, len(arr)):
    count_arr[arr[x]] = count_arr[arr[x]] + 1
minimal = -1
for x in range(0,5):
    if count_arr[x] > minimal:
	output = x
	minimal = count_arr[x]
print output 

File: bird.c
#include<stdio.h>

int main()
{
    long n,i,j,minimal;
    long count_arr[5];
    long temp, output;
    scanf("%ld", &n);
    for(j=0; j<5;j++)
	count_arr[j] = 0;
    for(i=0;i<n;i++)
    {
	scanf("%ld", &temp);
	count_arr[temp-1]++;
    }
    minimal = -1;
    for (j=0;j<5;j++)
    {
	if(count_arr[j] > minimal)
	{
	    output = j+1;
	    minimal = count_arr[j];
	}
    }
    printf("%ld\n", output);
}

